### PR TITLE
minor fixes to Kill Bill events and updates to other documents

### PR DESCRIPTION
**Pull request by reshmabidikar**
- Fixed document name in toc
- Minor changes to Kill Bill events document
- Added Information about some `metadata` objects to the Kill Bill events document
- Updated Notifications document to point to Kill Bill events document
- Made some improvements to the push notifications document

### CODE REVIEW

File: userguide/tutorials/kill_bill_events.adoc [hunk 1]
```diff
@@ -95,6 +95,32 @@ The following table provides details of all the events and their metadata.
...
+
+Let us now take a quick look at some of the metadata objects.
+
+=== SubscriptionMetadata
+
+The https://github.com/killbill/killbill-plugin-api/blob/d9eca5af0e37541069b1c608f95e100dbe13b301/notification/src/main/java/org/killbill/billing/notification/plugin/api/SubscriptionMetadata.java[SubscriptionMetadata] object is used to specify the metadata in case of the `SUBSCRIPTION_CREATION`, `SUBSCRIPTION_PHASE`, `SUBSCRIPTION_CHANGE`, `SUBSCRIPTION_CANCEL`, `SUBSCRIPTION_UNCANCEL` and `SUBSCRIPTION_BCD_CHANGE` events.
+
+It has the following fields:
+
+* `actionType` : Specifies whether the subscription is `EFFECTIVE` (subscription created with the current date) or `REQUESTED` (subscription created with a future date).
```
Comment (pierre) [inline userguide/tutorials/kill_bill_events.adoc]: It's not just `created` -- it's wether the event (creation, change, phase, etc.) is effective right now or is being scheduled (_requested_).

**Review by pierre — APPROVED** [review]
Looks great! Just one small comment.

File: userguide/tutorials/kill_bill_events.adoc (hunk 1 above)
Comment (reshmabidikar) [reply userguide/tutorials/kill_bill_events.adoc]: Fixed

**Comment by pierre** [comment]
🎉

**Pull request merged**